# Generated by Django 4.0.10 on 2024-05-10 00:52

from django.db import migrations, transaction
from django.db.utils import IntegrityError

@transaction.atomic
def change_site_domain(apps, schema_editor):
    Site = apps.get_model('sites', 'Site')
    Syndicate = apps.get_model('app', 'Syndicate')
    try:
        site, created = Site.objects.get_or_create(domain='localhost:3334', defaults={'name': 'localhost:3334'})
        syndicate = Syndicate.objects.get(id=1)
        syndicate.sites.add(site)
    except (Site.DoesNotExist, IntegrityError):
        pass

class Migration(migrations.Migration):

    dependencies = [
        ('app', '0078_remove_email_unique_constraint'),
        ('sites', '0002_alter_domain_unique'),
    ]

    operations = [
        migrations.RunPython(change_site_domain, migrations.RunPython.noop),
    ]